account_invoice_margin/tests/test_account_invoice_margin.py

```diff
@@ -120,7 +120,13 @@ def test_invoice_refund(self):
                 active_ids=self.invoice.ids,
                 active_id=self.invoice.id,
             )
-            .create({"refund_method": "refund", "reason": "reason test create"})
+            .create(
+                {
+                    "refund_method": "refund",
+                    "reason": "reason test create".format,
```

Comment:
Why adding `format`?

Reply:
@pedrobaeza indeed... I have no clue how that appeared there. Must have been added by a pre-commit formatting (which also sounds unlikely). I removed it now.